src/client/src/rt-system/logger.ts

```diff
@@ -8,7 +8,7 @@ enum levels {
 
 type SEVERITY = 'info' | 'warn' | 'error' | 'debug'
 
-let currentLevel = levels.verbose
+let currentLevel = levels.error
```

Comment:
I think this should be verbose in dev and warn in production.

Reply:
let currentLevel = process.env.NODE_ENV === 'production' ? levels.warn : levels.verbose

Reply:
Good catch, this got swept up into this PR by mistake.